pixart.js

```diff
@@ -1 +1,43 @@
+console.log('script.js loaded...');
+//globals
+var pixelForm = document.getElementById('form');
+var colorBox = document.querySelector('.brush');
+var colorInput = document.getElementById('color-field');
+var colorButton = document.getElementById('set-color');
 
+//create 20 div's with class .square and append them to document.body
+for (var i = 2; i < 8000; i++) {
```

Comment:
I think in this case it should still be `var i = 0; ` in your for loop! I'm assuming you might have done this because there are two additional divs already in our index.html file. So, if you wanted to check in your browser console, the following should be true: 

```
document.querySelectorAll("div").length === 8002
document.querySelectorAll(".square").length === 8000
```